### PR TITLE
Add ulid

### PR DESCRIPTION
**Pull request by SuperPaintman**
Hi! I added my [ulid](https://github.com/SuperPaintman/ulid) port for Crystal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [stumpy_gif](https://github.com/l3kn/stumpy_gif) - Write (animated) GIF images
  * [stumpy_png](https://github.com/l3kn/stumpy_png) - Read and write PNG images
  * [syscall.cr](https://github.com/kubo39/syscall.cr) - Raw syscall interface
+ * [ulid](https://github.com/SuperPaintman/ulid) - Universally Unique Lexicographically Sortable Identifier (ULID) in Crystal
```
Comment (veelenga) [inline README.md]: Please remove "in Crystal" part. See [CONTRIBUTING.md](https://github.com/veelenga/awesome-crystal/blob/master/CONTRIBUTING.md)

Comment (SuperPaintman) [inline README.md]: @veelenga oh, sorry. I fixed it

**Comment by veelenga** [comment]
Thank you @SuperPaintman

**Comment by SuperPaintman** [comment]
@veelenga thanks

**Pull request merged**